src/app/components/views/login-view/login-view.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { CookieService } from 'ngx-cookie-service';
+
+@Component({
+  selector: 'app-login-view',
+  templateUrl: './login-view.component.html',
+  styleUrls: ['./login-view.component.scss']
+})
+export class LoginViewComponent implements OnInit {
+
+  public uidForm!: FormGroup; // The form for logging in to ABC
+
+  constructor(private cookieService: CookieService, private router: Router) { }
+
+  ngOnInit(): void {
+
+    this.uidForm = new FormGroup({
```

Comment:
`this.uidForm = createUidForm()` here instead...and that method you create should return the form to assign to this variable.  dont get in the habit of adding code directly into onInit